### ISSUE TITLE
Fix accessibility of uberbar search feature in manager (accessibility defect)

### ISSUE THREAD

**Issue by paulbohman** [modxcms/revolution#12156]
For compliance with Web Content Accessibility Guidelines (WCAG)

Form label:

The search field needs a &lt;label&gt; tag, preferably one that is visible (the placeholder is not considered an accessible label because it disappears as soon as the keyboard focus enters the field), but you could also implement a hidden label in this particular case, because the magnifying glass conveys the idea of search (as long as the magnifying glass is turned into an actual submit button and as long as it has text that says "search").

Submit button is missing:

Ideally, the form would have an actual submit button so that blind users know they can submit the form. Blind users don't see the magnifying glass icon. In fact, the magnifying glass icon could be turned into a submit button, and you could maintain the same basic look as the exiting interface.

See accessibility to-do list at https://dequeuniversity.com/modx-accessibility.html#uberbar

**Comment by rthrash** [comment]
The accessibility project is now fully open for all, and this issue is being tracked/addressed there. 

Also the same applies to others and therefore closes #12142, closes #12143, closes #12144, closes #12145, closes #12146, closes #12146, closes #12147, closes #12148, closes #12149, closes #121450, closes #121451, closes #121452, closes #121453, closes #121454, closes #121455.

For a list of currently open issues associated withe the [MODX Accessible CMS Project](http://a11y.modx.com) see https://github.com/modxcms/a11y/issues/
